Bug: Fixes Inconsistent Background Coverage and Unwanted Inner Scrollbars on the Request Access Page After a Request Invite Is Submitted

nur-hasin opened the pull request:
# Description

Fixes inconsistent background coverage and unwanted inner scrollbars on the Request Access page after a request invite is submitted. This change ensures the background image covers the full viewport in both pre-submit and post-submit states and removes internal scroll containers that previously caused clipping and double-scroll behavior on mobile and tablet screens. Changes are scoped to the Request Access page styles and component only.


## Changes Made

- Ensured the Request Access page background always fills the viewport by enforcing full-viewport layout rules and consistent container sizing across both pre-submit and post-submit states.
- Eliminated nested/internal scroll areas that produced vertical scrollbars on small screens by centralizing scroll behavior to the page level and tightening overflow handling for child elements.
- Removed or adjusted fixed margins, paddings and fixed-size elements that previously caused horizontal and vertical overflow on narrow viewports.
- Constrained hero/illustration sizing using viewport-relative limits, so images no longer push content beyond the visible area on small devices.
- Added tuned responsive constraints for mobile and tablet breakpoints to prevent overflow while preserving design fidelity on larger screens.

## Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore (refactoring, documentation, or other non-functional changes)

## Related Issue

Closes #188 

## Screenshots

_desktop (>960px)_
<img width="1907" height="921" alt="Screenshot 2025-10-31 045846" src="https://github.com/user-attachments/assets/d214900d-9b49-421a-8b17-364e575a3ffb" />

 _tablet (520px-959px)_
<img width="1195" height="922" alt="Screenshot 2025-10-31 045920" src="https://github.com/user-attachments/assets/6137a80a-27f7-4c14-bdf6-952652191047" />

_mobile (320px-519px)_
<img width="647" height="927" alt="Screenshot 2025-10-31 045952" src="https://github.com/user-attachments/assets/18525db7-d4a8-4668-aefb-382b90412b98" />

## Checklist:

- [x] I have read the [**Contributing Guidelines**](/CONTRIBUTING.md).
- [x] My code follows the [**Code Style**](/docs/contributing/code-style.md) of this project.
- [ ] I have added tests that prove my fix is effective or that my feature works. See [**Testing Guidelines**](/docs/contributing/testing.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.

nur-hasin committed:
fix(request-access): preserve full-viewport background and remove inner scrollbars

motirebuma commented on client/src/components/RequestAccess/PersonalForm/PersonalForm.jsx:
```diff
@@ -29,164 +29,159 @@ const PersonalForm = (props) => {
+              <img
+                alt="Personal Plan"
+                src="/assets/PersonalPlan.png"
+                className={classes.image}
```
Line 54 references classes.image, but in `requestAccessStyles.jsx`, the image style is only defined inside the `@media (max-width: 520px)` block (lines 172-175). This means the image will be **unstyled** on screens wider than `520px`.

_Add a top-level image style definition in requestAccessStyles.jsx:_
```
image: {
    maxWidth: '100%',
    height: 'auto',
    objectFit: 'contain',
},
```

motirebuma reviewed (commented):
The PR addresses **the scrollbar issue mentioned in the title**, but contains few bugs that need to be fixed. Once these issues are resolved (missing image style, incorrect maxWidth, and invalid media query syntax), this should be ready to merge. **The core logic and structure improvements are good**. the issues are primarily styling/configuration problems.

### Summary
Must add: image style definition (somewhere before line 166, probably after inputContainer around line 152)

**Must fix**: 
Line 177 →  change '50px' to '90%' (to something appropriate)
**Should fix:** 
Lines 28, 41, 48, 53, 74, 86, 94 
fix typos (`fontsize` → `fontSize`, `letterspacing` → `letterSpacing`)

_The root issue is lines 166-180: root-level media queries are not supported by Material-UI's `makeStyles`. Media queries must be nested inside the style objects themselves._

**After fixing the above issues its ready to be merged.**

@flyblackbox @nur-hasin

nur-hasin committed:
fix: resolve styling issues in request access page

- Add missing top-level image style definition for proper rendering on all screens
- Fix opaqueBackground maxWidth from 50px to 80% to prevent mobile UI breakage
- Correct font property typos (fontsize->fontSize, letterspacing->letterSpacing)
- Move root-level media queries inside style objects for Material-UI compatibility
- Ensure consistent background coverage and eliminate mobile scrollbars
